packages/playground/website/src/components/layout/index.tsx

```diff
@@ -35,6 +35,15 @@ import {
 
 acquireOAuthTokenIfNeeded();
 
+export const modalSlugs = {
```

Comment:
lovely change!

Reply:
Since these constants are in place now, could the entire if/else clause below use them instead of string literals?

```
	if (currentModal === 'log') {
		return <LogModal />;
	} else if (currentModal === 'error-report') {
		return <ErrorReportModal blueprint={blueprint} />;
	} else if (currentModal === 'start-error') {
```